apps/api/src/app/environments-v1/usecases/output-renderers/in-app-output-renderer.usecase.ts

```diff
@@ -41,30 +41,34 @@ export class InAppOutputRendererUsecase {
     };
   }
 }
-const RedirectTargetEnumSchema = z.enum(['_self', '_blank', '_parent', '_top', '_unfencedTop']);
+const RedirectTargetEnumSchema = z.enum(['_self', '_blank']); // Example enum, replace with your actual enum
 
 const InAppRenderOutputSchema = z.object({
   subject: z.string().optional(),
   body: z.string(),
   avatar: z.string().optional(),
-  primaryAction: z
+  buttonSettings: z
```

Comment:
Ditto about sticking to the inAppOutputSchema as defined in @novu/framework

Reply:
Thats a bed naming, the output schema stayed the same, thats referring to the control schema